source/_components/sensor.speedtest.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: "Speedtest.net"
+description: "How to integrate Speedtest.net within Home Assistant."
+date: 2016-02-12 9:06
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: speedtest.png
+ha_category: Sensor
+featured: false
+---
+
+The `speedtest` sensor component uses the [Speedtest.net](https://speedtest.net/) web service to measure network bandwidth performance.
+
+By default, it will run every hour on the hour.  The user can change the update frequency in the config by defining the minute, hour, and day for speedtest to run.
+
+To add speedtest.net sensors to your installation, add the following to your `configuration.yaml` file:
+
+Once per hour, on the hour (default):
+```yaml
+sensor:
+  platform: speedtest
+  monitored_conditions:
+    - ping
+    - download
+    - upload
+```
+
+More examples:
+
+Every half hour of every day:
+```yaml
+sensor:
+  platform: speedtest
+  minute:
+    - 0
+    - 30
+  monitored_conditions:
+    - ping
+    - download
+    - upload
+```
+
+Everyday at 12:30AM, 6:30AM, 12:30PM, 6:30PM:
+```yaml
+sensor:
+  platform: speedtest
+  minute: 30
+  hour:
+    - 0
+    - 6
+    - 12
+    - 18
+  monitored_conditions:
+    - ping
+    - download
+    - upload
+```
+
+Configuration variables:
+
+- **monitored_conditions** array (*Required*): Conditions to display in the frontend.
+  - **ping**: A human-readable text summary.
+  - **download**: The type of precipitation occurring.
+  - **upload**: The average expected intensity of precipitation occurring.
```

Comment:
Ping, download and upload descriptions do not seem to be accurate. Unless bandwidth is somehow related to precipitation. :wink: